platform/linux-generic/arch/aarch64/odp_crypto_armv8.c

```diff
@@ -449,6 +476,261 @@ static int process_aes_gcm_param(odp_crypto_generic_session_t *session)
 	return 0;
 }
 
+static void
+aes_cbc_encrypt_hmac_gen(odp_packet_t pkt,
+			 const odp_crypto_packet_op_param_t *param,
+			 odp_crypto_generic_session_t *session)
+{
+	armv8_cipher_digest_t arg;
+	void *iv_ptr;
+	uint32_t in_len = param->cipher_range.length;
+	uint32_t hash_len = param->auth_range.length;
+	uint32_t in_pos = param->cipher_range.offset;
+	uint32_t hash_pos = param->auth_range.offset;
+	uint8_t key_expanded[256] = {0};
+	uint8_t hash_buf[hash_len];
+	uint8_t digest[ARM_CRYPTO_MAX_DIGEST_LENGTH];
+	int rc;
+
+	/* Fail early if cipher_range is too large */
+	if (odp_unlikely(in_len > ARM_CRYPTO_MAX_DATA_LENGTH)) {
+		ODP_DBG("ARM Crypto: Packet size too large for requested operation\n");
```

Comment:
It is not the packet size that is too large but the cipher range, as also mentioned in the comment. The "for requested operation" part of the message seems unnecessary. Maybe have it simply: "Cipher range too large".